src/factories/series/LineSeriesContainer.ts

```diff
@@ -0,0 +1,30 @@
+module n3Charts.Factory.Series {
+  'use strict';
+
+  export class LineSeriesContainer extends Utils.BaseFactory {
+
+    public svg: D3.Selection;
+
+    constructor() {
+      super();
+    }
+
+    create() {
+      this.createContainer(this.factoryMgr.get('container').vis);
```

Comment:
basically just rename `Factory.Series.LineSeriesContainer` to `Factory.Series.Line` and not use the `Factory.Container.vis` property but some data container